StitchCore/Tests/stitch-ios-sdk-protoTests/Internal/Common/SynchronizedTests.swift

```diff
@@ -7,6 +6,7 @@ class SynchronizedTests: XCTestCase {
         fileprivate var value = 0
     }
 
+    @available(OSX 10.12, *)
```

Comment:
ah this was the same error I was getting that randomly disappeared when I upgraded to high sierra. I'm fine leaving this in here but is it fine that our test here won't run on linux?